pkg/codegen/internal/test/testdata/provider-config-schema/schema.json

```diff
@@ -23,10 +23,7 @@
         "type": "string",
         "description": "This is a huge secret",
         "defaultInfo": {
```

Comment:
Any special reason for these formatting changes? Removing them would make review easier :)

Reply:
Sorry. That was a "format on save" behavior. It won't happen again. I can undo it if desired. 

As an aside: how do you do code reviews that involve programatic changes to 100+ files.

Reply:
NP at all.. It's a tiny thing. If we agree on the spec of the right "format on save" to use that'd be nice, reformat and avoid the noise.